Cargo.toml

```diff
@@ -8,7 +8,9 @@ authors = [
 ]
 
 [dependencies]
-piston_window = "0.65.0"
+piston = "0.32.0"
+piston2d-graphics = "0.21.1"
 piston2d-opengl_graphics = "0.43.0"
+pistoncore-glutin_window = "0.36.0"
```

Comment:
I'm a little torn here, since it was nice that `piston_window` tracked all three dependencies together and we didn't need to look up and bump them all together manually.